packages/client-core/src/components/InviteToast/index.tsx

```diff
@@ -27,58 +27,54 @@ import React, { useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { UserName } from '@ir-engine/common/src/schema.type.module'
-import capitalizeFirstLetter from '@ir-engine/common/src/utils/capitalizeFirstLetter'
-import { Button } from '@ir-engine/editor/src/components/inputs/Button'
+
 import { useMutableState } from '@ir-engine/hyperflux'
 
+import Button from '@ir-engine/ui/src/primitives/tailwind/Button'
+import { first } from 'lodash'
+import { HiMiniXMark } from 'react-icons/hi2'
 import { InviteService, InviteState } from '../../social/services/InviteService'
 import { AuthState } from '../../user/services/AuthService'
-import styles from './index.module.scss'
 
 const InviteToast = () => {
+  const { t } = useTranslation()
   const inviteState = useMutableState(InviteState)
   const authState = useMutableState(AuthState)
-  const newestInvite =
-    inviteState.receivedInvites.total.value > 0 ? inviteState.receivedInvites.invites[0].value : ({} as any)
-  const { t } = useTranslation()
+  const newestInvite: any = first(inviteState.receivedInvites.invites)?.value
```

Comment:
can you define the type of `newestInvite` here?

Reply:
done 15368c97989cd7ea379f16095db878b69f57142e